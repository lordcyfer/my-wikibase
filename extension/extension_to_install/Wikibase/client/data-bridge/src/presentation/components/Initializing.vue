<template>
	<div class="wb-db-init">
		Loadingâ€¦
	</div>
</template>

<script lang="ts">
import Vue from 'vue';
import Component from 'vue-class-component';

@Component
export default class Initializing extends Vue {}
</script>

<style lang="scss">
.wb-db-init {
	@include dialogBody();
}
</style>
