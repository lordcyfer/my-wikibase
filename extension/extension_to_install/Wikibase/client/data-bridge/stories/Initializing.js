import { storiesOf } from '@storybook/vue';

import Initializing from '@/presentation/components/Initializing';

storiesOf( 'Initializing', module )
	.add( 'loading view', () => ( {
		components: { Initializing },
		template: '<Initializing />',
	} ), { info: true } );
