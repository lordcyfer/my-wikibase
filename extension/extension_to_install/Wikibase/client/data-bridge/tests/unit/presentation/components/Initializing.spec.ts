import Initializing from '@/presentation/components/Initializing.vue';
import { shallowMount } from '@vue/test-utils';

describe( 'Initializing', () => {
	it( 'is a Vue instance', () => {
		const wrapper = shallowMount( Initializing );
		expect( wrapper.isVueInstance() ).toBeTruthy();
	} );

	it( 'renders correctly', () => {
		const wrapper = shallowMount( Initializing );
		expect( wrapper.element ).toMatchSnapshot();
	} );

} );
