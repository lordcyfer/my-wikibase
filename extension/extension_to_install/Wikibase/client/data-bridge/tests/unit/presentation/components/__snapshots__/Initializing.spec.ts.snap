// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Initializing renders correctly 1`] = `
<div
  class="wb-db-init"
>
  
	Loadingâ€¦

</div>
`;
